docs/source/markdown/podman-pull.1.md

```diff
@@ -21,42 +21,24 @@ Images are stored in local image storage.
 
 ## SOURCE
 
- The SOURCE is the location from which the container images are pulled.
- The Image "SOURCE" uses a "transport":"details" format.  Only the `docker` (container registry)
- transport is allowed for remote access.
+ SOURCE is the location from the container image is pulled from. It supports all transports from `containers-transports(5)`. If no transport is specified, the `docker` (i.e., container registry) transport is used.  For remote clients, `docker` is the only supported tranport.
```

Comment:
It’s not _quite_ true that `docker` is used without a transport specification — and it’s also not at all untrue: non-`docker` names trigger short name / alias search, so `busybox` and `docker://busybox` are not the same.

The page already talks about short names above; maybe mention this here.

Reply:
```suggestion
 SOURCE is the location from which the container image is pulled. It supports all transports from `containers-transports(5)`. If no transport is specified, the `docker` (i.e., container registry) transport is used.  For remote clients, `docker` is the only supported transport.
```